git_wrapper/__init__.py

```diff
@@ -4,4 +4,3 @@
 
 __author__ = """Jason Joyce"""
 __email__ = 'fuzzball81@gmail.com'
-__version__ = '0.2.8'
```

Comment:
Oh, is that no longer necessary? We may want to update the README file section on releasing to indicate that only the HISTORY file needs to be updated, then?

I wonder if it would be worth encouraging new features to come with a HISTORY update from now on too, so we'll only need to tag to prepare a new release rather than having to submit a patch first? If I understand correctly. Maybe writing up the highlights all at once is fine.

Reply:
The plan here is to use setuptools_scm, which automatically generates the version based on the git tag.  So, when we do a release, the publish job should have a version of the project checked out by tag for newest release and build from that.  I tested this locally, and it seemed to work as expected.  If you do a build without a tag, it will build with <most-recent-tag>.dev<#>+<git hash> so you can install a dev build easily.  
For now, yes, only the HISTORY file would need to be updated for a release, after tagging.  I would like to make this more automated as well, though the other option would be to do a more curated changelog that describes features and bugs fixed, rather than simply listing commits between releases.  It is possible we may even want both, I am not really sure yet.  Either way, I propose we consider this a future enhancement, otherwise I am afraid I'll never finish this patch.  And yes, I'll update the relevant section of the readme in an update shortly.